nixos/modules/virtualisation/nixos-containers.nix

```diff
@@ -494,7 +494,7 @@ in
               '';
               type = lib.mkOptionType {
                 name = "Toplevel NixOS config";
-                merge = loc: defs: (import "${toString config.nixpkgs}/nixos/lib/eval-config.nix" {
```

Comment:
> 
> , also considering a completely different version can be loaded with the nixpkgs option.

Am I reading this wrong or isn't `config` the host `config`, and `config.nixpkgs` same nixpkgs as host nixpkgs?